lib/devices_api/signin.ex

```diff
@@ -0,0 +1,23 @@
+defmodule DevicesApi.Signin do
+  @moduledoc """
+  Delegate functions to handle Signin context
+  """
```

Comment:
Aqui, a meu ver, deveríamos ter uma documentação do contexto/domínio. 

Documentação de módulo não é só para descrever "o que está implementado", mas também "o que queremos que a implementação atinja". 

Por exemplo aqui poderíamos ter algo como:

```suggestion
  @moduledoc """
  Sign in domain.
  
  Sign in is the process of exchanging credentials for a session. Current approach is to have JWT compact tokens in their JWS variant.
  
  A token contains information like subject, issuer, etc. See `xxxx` for more information.
  
  Sign ins generate audit records. Many consecutive failed attempts will lock the user.
  """
```